consoles/VNC.pm

```diff
@@ -878,7 +880,11 @@ sub _receive_message {
     my $self = shift;
 
     my $socket = $self->socket;
-    $socket or die 'socket does not exist. Probably your backend instance could not start or died.';
+    #$socket or die 'socket does not exist. Probably your backend instance could not start or died.';
```

Comment:
should be deleted then but I would like to keep the text in the exception error message. We should give that hint to the test writer that probably we should not have run into that step at all, i.e. test code need to be changed or the SUT does not respond. And not "there is no socket because something is wrong in the backend"

Reply:
Fixed in https://github.com/os-autoinst/os-autoinst/pull/864